generatorcli.py

```diff
@@ -51,11 +52,12 @@ def find_edge_by_nodes(_node_a, _node_b):
 
             if find_node_with_identifier(identifier) is None:
                 is_valid = True
-                node = Node(identifier, x, y, desc)
+                node = Node(uuid=identifier)
+                node.geo_node = GeoNode(x, y)
                 nodes.append(node)
             else:
                 print(f"Node with id {identifier} already exists. Please use a different id.")
-        elif re.match(r'edge [a-zA-Z_0-9]+ [a-zA-Z_0-9]+ (coords (-?\d+(\.\d+)?,-?\d+(\.\d+)?)+)?', command):
+        elif re.match(r'edge [a-zA-Z_0-9]+ [a-zA-Z_0-9]+(coords (-?\d+(\.\d+)?,-?\d+(\.\d+)?)+)?', command):
```

Comment:
This space needs to be moved in front of the `coords` word but inside the brackets. Otherwise the correct command will be `edge n0 n1coords ...`

Reply:
Wait, this file will be removed anyway... So more a thought for the other repository.